src/os/inc/osapi-version.h

```diff
@@ -44,14 +47,19 @@
 #define OS_MINOR_VERSION 0  /*!< @brief Minor version number */
 #define OS_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
CFE_REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.